app/models/connector/package.scala

```diff
@@ -79,6 +79,16 @@ package object connector {
     }
   }
 
+  def lossClaimReads[A: Reads](implicit logger: Logger): HttpReads[ApiResponse[A]] = (method: String, url: String, response: HttpResponse) => {
```

Comment:
I discussed this with Simon last week. He requested a tailored approach for the loss claims to ensure we cover all bases. Specifically, he wanted me to explicitly list all the error responses for the Create that would come from the backend: BAD_REQUEST, NOT_FOUND, CONFLICT, and UNPROCESSABLE_ENTITY, rather than relying on a catch-all.